src/frontend/src/components/RecordSearch/Record/RecordSummary/CaseBalances.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import BalanceList from "./BalanceList";
+import { CaseData } from "../types";
+import currencyFormat from "../../../../service/currency-format";
+
+interface Props {
+  cases?: CaseData[];
+  totalBalance: number;
+}
+
+const hasBalanceDue = (element: CaseData) => {
+  return element.balance_due > 0;
+};
+
+export default class CaseBalances extends React.Component<Props> {
+  render() {
+    if (!this.props.cases) {
+      return null;
+    }
+
+    const listItems = this.props.cases
+      .filter(hasBalanceDue)
+      .map((element: CaseData, i: number) => {
+        return (
+          <li className="mb2" key={i}>
+            <a
+              className="link underline hover-blue break-all"
+              href={`#${element.case_number}-1`}
```

Comment:
Is there a reason we aren't just linking to the case?

Reply:
@wittejm's initial comment on the PR mentioned to use the in page link so that is what I went with but I just noticed this would actually be linking to the first charge so I will update that. If we instead wanted to link to the actual case details page I can look at making that change as well.